routes/movies.routes.js

```diff
@@ -0,0 +1,35 @@
+const router = require("express").Router();
+const Movies = require('/models/movies.model')
+const Celebrities = require('/models/celebrities.model')
```

Comment:
Here, the error on the terminal was here,
You should write like this: 
const Movie = require("../models/movies.model")
const Celebrity = require("../models/celebrities.model")